lib/WP_Auth0_CustomDBLib.php

```diff
@@ -1,7 +1,41 @@
 <?php
-
+/**
+ * Contains class WP_Auth0_CustomDBLib
+ *
+ * @package WP-Auth0
+ *
+ * @since 2.0.0
+ */
+
+/**
+ * Class WP_Auth0_CustomDBLib
+ */
 class WP_Auth0_CustomDBLib {
 
+	/**
+	 * Get JS to use in the custom database script.
+	 *
+	 * @param string $name  - Database script name.
+	 * @param string $token - Migration token.
+	 *
+	 * @return bool|string
+	 */
+	public static function get_script( $name, $token ) {
+		$script = (string) file_get_contents( WPA0_PLUGIN_DIR . 'lib/scripts-js/db-' . $name . '.js' );
```

Comment:
Whenever I see `file_get_contents` followed by a variable. 👀

Reply:
https://secure.php.net/manual/en/function.file-get-contents.php

> This function is similar to file(), except that file_get_contents() returns the file in a string, starting at the specified offset up to maxlen bytes. On failure, file_get_contents() will return FALSE.

Worst that happens is a PHP warning if the file does not exist.